### DISCUSSION TITLE
Nginx Proxy Manager - serves paperless dummy certificate

### DISCUSSION THREAD

**Discussion by rdrr** [jonaswinkler/paperless-ng#1608] [Support]
I think I've got nginx proxy manager (NPM) setup, through the GUI, in the correct manner that the paperless-ng docs recommend nginx as a reverse proxy.  I've put the majority of the nginx config in the custom location.  Whenever I add proxy_pass http://IP:8000, the status in NPM goes to offline and when I navigate to the url it's not using my Let's Encrypty cert that I was able to obtain through NPM, but it's using a dummy cert issued by NPM.  When I leave off proxy_pass, I get a combo of 520, 502, and 400 based on how I change http or https in the location.

Anyone have a working setup with NPM?

**Comment by NiFNi** [comment]
Yes. Working fine for me with just the defaults. Websocket support should be switched on. Nothing in advanced.

**Reply by Chiller2019** [reply]
And with which settings on "Edit Proxy Host" -> "Advanded"?

**Reply by NiFNi** [reply]
As I wrote, nothing in there.

**Reply by Chiller2019** [reply]
> socket support should b

Sorry, my fault, dont read to the end :)
